skore/tests/unit/project/test_load.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+import pytest
+from skore.project import Project
+from skore.project.load import _load
+
+
+@pytest.fixture
+def tmp_project_path(tmp_path):
+    """Create a project at `tmp_path` and return its absolute path."""
+    # Project path must end with ".skore"
+    project_path = tmp_path.parent / (tmp_path.name + ".skore")
+    os.mkdir(project_path)
+    os.mkdir(project_path / "items")
+    os.mkdir(project_path / "views")
+    return project_path
+
+
+def test_load_no_project():
+    with pytest.raises(FileNotFoundError):
+        _load("/empty")
+
+
+def test_load_absolute_path(tmp_project_path):
+    p = _load(tmp_project_path)
+    assert isinstance(p, Project)
+
+
+def test_load_relative_path(tmp_project_path):
+    os.chdir(tmp_project_path.parent)
+    p = _load(tmp_project_path.name)
+    assert isinstance(p, Project)
```

Comment:
```suggestion
@pytest.mark.parametrize("path_type, path_transform", [
    ("absolute_path", lambda path: path),
    ("relative_path", lambda path: path.name)
], ids=["absolute_path", "relative_path"])
def test_load_project(tmp_project_path, path_type, path_transform):
    if path_type == "relative_path":
        # move away from the working directory
        os.chdir(tmp_project_path.parent)
    p = _load(path_transform(tmp_project_path))
    assert isinstance(p, Project)
```

Reply:
This looks a bit overengineered to me actually